apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/(commit)/documents/[documentUuid]/_components/MetadataInfoTabs/index.tsx

```diff
@@ -21,6 +21,13 @@ export const MetadataInfoTabs = forwardRef<HTMLDivElement, Props>(
     ref,
   ) {
     const [selectedTab, setSelectedTab] = useState<string>('metadata')
+
+    useEffect(() => {
+      if (tabs.length > 0) {
+        setSelectedTab(tabs[0]!.value)
+      }
+    }, [tabs])
```

Comment:
Why this? What was selected before?

Reply:
if the tabs prop changes the selected tab didn't update which means it might be holding a value from a previous tabs prop that is no longer available